tools/gen_unicase.rb

```
@@ -0,0 +1,270 @@
+
+encoded = TABLES.map do |name, map, _|
+  runs = runs_of(map)
+  multi = multis_of(map).map { |cp, s| [cp, place.call(s), s.bytesize] }
+  [name, runs, multi]
+end
+
+widest = ([lower, upper, title, fold].flat_map { |m| m.values.map(&:bytesize) }).max
+
+# ------------------------------------------------------------------- emit
+
+def hex(cp) = "0x%05X" % cp
```

RuboCop (1.88.2)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)